### PR TITLE
Improve grammar of the WP CLI handbook

### PR DESCRIPTION
**Pull request by pmbaldha**

### CODE REVIEW

File: committers-credo.md [hunk 1]
```diff
@@ -16,7 +16,7 @@ Practically-speaking:
 * Given a choice between Good, Fast, and Cheap, we pick Good and Slow. Creating great software is a journey, not a destination.
 * Product decisions are made [in alignment with our core philosophy](https://make.wordpress.org/cli/handbook/philosophy/).
 * [Code review is an integral part of our development workflow](https://make.wordpress.org/cli/handbook/code-review/). Pull requests need approval of at least one other committer before merging. If you’d like more than one code review, please don’t hesitate to request one. You can request a review from a specific person, or from the `@wp-cli/committers` team.
-* If you need help with something, ask for help. Seriously, ask for help any time you feel like you need help. No sense getting stuck.
+* If you need help with something, ask for help. Seriously, ask for help any time you feel like you need help. No sense of getting stuck.
```
Comment (schlessera) [inline committers-credo.md]: This is grammatically wrong, the original sentence was correct. Please revert this change.

File: code-review.md [hunk 2]
```diff
@@ -42,7 +42,7 @@ Next, check for readability. Functions, variables, and files should be named cle
 
 As readability is inherently subjective, this requires the ability to look back from the immediate code changes to the bigger picture. Think of someone months down the road trying to trace a given code path through the current changeset. Are there unnecessary steps that could be simplified? Are code comments and inline documentation robust enough to recreate the thought process behind the code?
 
-Finally, check for elegance and overall quality. Code should follow existing and known patterns so that others can understand it at a glance. If a change introduces a chance to refactor surrounding functionality, to abstract and standardize old code into new patterns, suggest those opportunities. 
+Finally, check for elegance and overall quality. The code should follow existing and known patterns so that others can understand it at a glance. If a change introduces a chance to refactor surrounding functionality, to abstract and standardize old code into new patterns, suggest those opportunities.
```
Comment (schlessera) [inline code-review.md]: The original was correct and reads better. "Code" can be used as an uncountable subject. Please revert this change.

File: common-issues.md [hunk 3]
```diff
@@ -204,5 +204,5 @@ See also: [#587](https://github.com/wp-cli/wp-cli/issues/587)
 
 The `wp core update` command is designed to work for WordPress 3.4 and above. To be able to update an older website to latest WordPress, you could try one of the following alternatives:
 
-1. **Fully-automated:** Run `wp core download --force` to download latest WordPress and replace it with your files (don't worry, `wp-config.php` will remain intact). Then, run `wp core update-db` to update the database. Since the procedure isn't ideal, run once again `wp core download --force` and the new version should be available.
+1. **Fully-automated:** Run `wp core download --force` to download the latest WordPress and replace it with your files (don't worry, `wp-config.php` will remain intact). Then, run `wp core update-db` to update the database. Since the procedure isn't ideal, run once again `wp core download --force` and the new version should be available.
```
Comment (schlessera) [inline common-issues.md]: This is grammatically wrong, the original sentence was correct. Please revert this change.

File: documentation-standards.md [hunk 4]
```diff
@@ -1,6 +1,6 @@
 # Documentation Standards
 
-To promote similarity and consistency between WP-CLI commands, we've produced these documentation standards we'd encourage you to follow. Commonality is a key component of usability, because it reduces the mental overhead required when switching between commands.
+To promote similarity and consistency between WP-CLI commands, we've produced these documentation standards we'd encourage you to follow. The commonality is a key component of usability, because it reduces the mental overhead required when switching between commands.
```
Comment (schlessera) [inline documentation-standards.md]: This is grammatically wrong, the original sentence was correct. Please revert this change.

File: force-output-specific-locale.md [hunk 5]
```diff
@@ -1,6 +1,6 @@
 # Force output to a specific locale
 
-WP-CLI always outputs English because it doesn't support localization. But, because WordPress supports localization, you may see non-English output when performing specific commands.
+WP-CLI always outputs English because it doesn't support localization. But, because WordPress supports localization, you may see the non-English output when performing specific commands.
```
Comment (schlessera) [inline force-output-specific-locale.md]: This conveys the wrong meaning, the original sentence was correct. Please revert this change.

File: troubleshooting.md [hunk 6]
```diff
@@ -2,7 +2,7 @@
 
 Before you start to report a new issue on the GitHub repository, make sure to check your local installation, as some settings may typically result in an unexpected outcome of your WP-CLI commands.
 
-### How do I get more verbose informations about my WP-CLI installation?
+### How does I get more verbose informations about my WP-CLI installation?
```
Comment (schlessera) [inline troubleshooting.md]: This is grammatically wrong, the original sentence was correct. Please revert this change.

File: contributing.md [hunk 7]
```diff
@@ -30,7 +30,7 @@ Want to contribute a new feature? WP-CLI is a mature project, and already chock-
...
+Once you've decided to commit the time to see your pull request through, please [follow our guidelines for creating a pull request](https://make.wordpress.org/cli/handbook/pull-requests/) to make sure it's a pleasant experience. See "[Setting up](https://make.wordpress.org/cli/handbook/pull-requests/#setting-up)" for details on making local modifications to WP-CLI. Keep in mind pull requests are [expected to have tests](https://make.wordpress.org/cli/handbook/pull-requests/#running-and-writing-tests) covering the scope of the change. Read through [our code review guidelines](https://make.wordpress.org/cli/handbook/code-review/) for a better understanding of how your pull request will be evaluated.
```
Comment (schlessera) [inline contributing.md]: This is grammatically wrong, the original sentence was correct. Please revert this change.

File: committers-credo.md (hunk 1 above)
Comment (pmbaldha) [reply committers-credo.md]: @schlessera Done

File: code-review.md (hunk 2 above)
Comment (pmbaldha) [reply code-review.md]: @schlessera Done

File: documentation-standards.md (hunk 4 above)
Comment (pmbaldha) [reply documentation-standards.md]: @schlessera Done

File: force-output-specific-locale.md (hunk 5 above)
Comment (pmbaldha) [reply force-output-specific-locale.md]: @schlessera Done

File: contributing.md (hunk 7 above)
Comment (pmbaldha) [reply contributing.md]: @schlessera Done

File: troubleshooting.md (hunk 6 above)
Comment (pmbaldha) [reply troubleshooting.md]: @schlessera Done

File: common-issues.md (hunk 3 above)
Comment (pmbaldha) [reply common-issues.md]: @schlessera Done
